tools/apprunner-maven-plugin/src/main/java/com/dremio/nessie/quarkus/maven/QuarkusApp.java

```diff
@@ -62,7 +67,8 @@ private QuarkusApp(RunningQuarkusApplication runningApp) {
    * @throws MojoExecutionException if an error occurs during execution
    */
   public static QuarkusApp newApplication(MavenProject project, RepositorySystem repoSystem,
-      RepositorySystemSession repoSession, String appArtifactId)
+      RepositorySystemSession repoSession, String appArtifactId, Properties applicationProperties,
+      Collection<Path> additionalDependencyPaths)
```

Comment:
what are the additional dependencies for?

Reply:
The `MojoSourceConfig` needs to be part of the application runtime in order to be able to set the properties. Should probably find a better/smarter way of (it's still a bit experimental)